specification/managedservices/resource-manager/Microsoft.ManagedServices/preview/2022-01-01-preview/managedservices.json

```diff
@@ -0,0 +1,1298 @@
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns a list of operations.",
+            "schema": {
+              "$ref": "#/definitions/OperationList"
+            }
+          },
+          "default": {
+            "description": "The error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get Registration Operations": {
+            "$ref": "./examples/GetOperations.json"
+          }
+        }
+      }
+    },
+    "/{scope}/providers/Microsoft.ManagedServices/operations": {
+      "get": {
+        "tags": [
+          "Operations"
+        ],
+        "operationId": "OperationsWithScope_List",
+        "description": "Gets a list of the operations with the scope.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ScopeParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns a list of operations.",
+            "schema": {
+              "$ref": "#/definitions/OperationList"
+            }
+          },
+          "default": {
+            "description": "The error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get Registration Operations": {
+            "$ref": "./examples/GetOperationsWithScope.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "RegistrationDefinition": {
+      "properties": {
+        "properties": {
+          "description": "The properties of a registration definition.",
+          "$ref": "#/definitions/RegistrationDefinitionProperties"
+        },
+        "plan": {
+          "description": "The details for the Managed Services offer’s plan in Azure Marketplace.",
+          "$ref": "#/definitions/Plan"
+        },
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The fully qualified path of the registration definition."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The type of the Azure resource (Microsoft.ManagedServices/registrationDefinitions)."
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The name of the registration definition."
+        },
+        "systemData": {
+          "readOnly": true,
+          "description": "The metadata for the registration assignment resource.",
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData"
```

Comment:
>v2 [](http://example.com/codeflow?start=67&length=2)

Any reason you aren't using v3 for this?

Reply:
No reason, just reused that reference from an example. Changed it to v3, thanks